src/styles/styles.js

```diff
@@ -1092,6 +1092,20 @@ const styles = {
         height: 24,
         lineHeight: 20,
     },
+
+    iouAmountText: {
+        fontFamily: fontFamily.GTA_BOLD,
+        fontWeight: fontWeightBold,
+        fontSize: variables.iouAmountTextSize,
```

Comment:
@shawnborton perhaps you could recommend a more standard name than `iouAmountTextSize `, as we might end up using this size again?

Reply:
I'm waiting for @shawnborton's view here 👍

Reply:
No need to block the issue for this change. We can refactor it once its being reused.